cypress/integration/SelectLabs.spec.ts

```diff
@@ -366,4 +352,284 @@ describe('Select', () => {
       });
     });
   });
+
+  context(`given the "Scrollable" story is rendered`, () => {
+    beforeEach(() => {
+      h.stories.load('Labs|Select/React/Top Label', 'Scrollable');
+    });
+
+    context('when the select button is focused', () => {
+      beforeEach(() => {
+        cy.findByLabelText('Label').focus();
+      });
+
+      context(
+        'when a character is typed (provided no other characters have been typed in the last 500ms), the select should select the first matching option beyond the currently selected option (cycling back to the beginning of the options if necessary)',
```

Comment:
I've added detailed descriptions for the typeahead and scrolling tests using `context`. The tests felt a bit random without this additional information -- it seemed unclear why we were testing certain things, whereas this additional context better conveys the intent behind our testing. Let me know if anyone thinks there's a better way to communicate this information.